parse/parse.go

```diff
@@ -40,6 +40,67 @@ var unwantedLinePrefixes = [][]byte{
 	[]byte("//go:generate genny "),
 }
 
+func subIntoLiteral(lit, t, specificType string) string {
+	var i int
+	var newLine string
+	for {
```

Comment:
I think now that this function is just replacing the occurrences in a single literal, this for loop can be avoided. Maybe just replacing the first occurrence with an if and then replacing all other occurrences with a single call to `Replace`.

Reply:
Totally right.  Nice!  I think this refactoring addresses your other issues/questions with this function as well.